dirs/dirs.go

```diff
@@ -183,7 +183,11 @@ func SetRootDir(rootdir string) {
 
 	SnapDataDir = filepath.Join(rootdir, "/var/snap")
 	SnapDataHomeGlob = filepath.Join(rootdir, "/home/*/", UserHomeSnapDir)
-	SnapAppArmorDir = filepath.Join(rootdir, snappyDir, "apparmor", "profiles")
+	if release.DistroLike("opensuse") {
```

Comment:
Yeah I was thinking about that but I never tried it on suse proper so I thought about using opensuse for now, which I at least use regularly.

Reply:
Have you checked to see if derivative distributions like GeckoLinux have `ID_LIKE=opensuse`? The only thing _actually_ required to identify as a member of the SUSE Linux family is `ID_LIKE=suse`, so...

Reply:
In retrospect, the correct answer here is to do `if !release.DistroLike("ubuntu")`. Your extended paths is Ubuntu-specific.

Reply:
I think that `release.DistroLike("suse")` is the way to go here. If I'm not mistaken, with partial confinement enabled, we'll end up using `/var/lib/snapd/apparmor/profiles` on Debian too.

Reply:
@bboozzoo Does Debian have the snappy-specific patch in their apparmor userspace tooling to use the other folder? Because I have a feeling most distros don't have that, so we'd probably want `!release.distroLike("ubuntu")` instead.

Reply:
Hm good question, @zyga ?

Reply:
@Conan-Kudo @bboozzoo debian 9 and 10 have support for snappy-specific profile locations.

Reply:
@zyga So this is only in the Debian family, then?

Reply:
@Conan-Kudo I don't know if it's _only_ there. Hopefully with the next apparmor userspace release the issue will become moot as those things can be configured by snapd then,